components/authentication-framework/org.wso2.carbon.identity.application.authentication.framework/src/main/java/org/wso2/carbon/identity/application/authentication/framework/handler/step/impl/DefaultStepHandler.java

```diff
@@ -897,6 +897,13 @@ protected String getRedirectUrl(HttpServletRequest request, HttpServletResponse
                 }
             }
         }
+        if (StringUtils.isBlank(reCaptchaParamString.toString())) {
+            String captchaParamString = (String) context.getProperty(FrameworkConstants.CAPTCHA_PARAM_STRING);
+            if (captchaParamString != null) {
+                reCaptchaParamString.append(captchaParamString);
```

Comment:
ReCaptchaParamString, captchaParamString variable names does not indicate the different. Shall we rename them. eg: captchaParamString -> captchaParamStringFromContext.

Reply:
Addressed in 75cdac0d66fb2eacead4372c14481e4fa6777ba2